server/src/main/java/org/elasticsearch/index/mapper/NestedObjectMapper.java

```diff
@@ -182,11 +182,6 @@ public MapperBuilderContext createChildContext(String name, Dynamic dynamic) {
                 getMergeReason()
             );
         }
-
```

Comment:
No need to override as it is taken care of via the superclass constructor / attribute